dali/base/dadfs.cpp

```diff
@@ -11759,6 +11760,13 @@ SecAccessFlags CDistributedFileDirectory::getFDescPermissions(IFileDescriptor *f
     return retPerms;
 }
 
+SecAccessFlags CDistributedFileDirectory::getDropZoneScopePermissions(const char *dropZoneNameOrIP,const char *dropZonePath,bool isPlane,IUserDescriptor *user,unsigned auditflags)
+{
+    StringBuffer scope;
+    makeDropZoneScope(scope,isPlane,dropZoneNameOrIP,dropZonePath);
```

Comment:
see comments on impl. re. using CDfsLogicalFileName,
e.g. this becomes:
```
CDfsLogicalFileName lfn;
lfn.setPlaneExternal(dropZoneName, dropZonePath);
return getScopePermissions(lfn.get(),user,auditflags);
```

Reply:
```
    CDfsLogicalFileName lfn;
    if (isPlane)
        lfn.setPlaneExternal(dropZoneNameOrIP, dropZonePath);
    else
        lfn.setExternal(dropZoneNameOrIP, dropZonePath);
    return getScopePermissions(lfn.get(),user,auditflags);
```

Reply:
yes something like that, I think it will further simplify when the dropzone plane name is resolved in the case when an host/ip is supplied ( as per prev. comments re. planes are auto-generated on baremetal for drop zones )